main.tex

```diff
@@ -954,17 +954,27 @@ \subsubsection{Unit Support}
 \subsection{Convolution}
 % Adam G.
 
-The \package{astropy.convolution} subpackage implements `normalized convolution' \citep[e.g.,][]{Knutsson1993}, which is an image reconstruction technique in which missing data are ignored during the convolution and replaced with values interpolated using the kernel.   An example of this is given in \figurename~\ref{fig:convolution-example}.  In versions $\leq 1.3$, the direct convolution and Fast Fourier Transform (FFT) convolution approaches were not consistent, with FFT convolution implementing normalized convolution and direct convolution implementing a different approach.  As of version 2.0, the two methods are consistent and include a suite of consistency checks.
+The \package{astropy.convolution} subpackage implements
+``normalized convolution'' \citep[e.g.,][]{Knutsson1993} that is an image
```

Comment:
Single quotes only make sense to me if it is a Python string, so I changed it to good ol' double quotes.